atlp-rwanda/e-commerce-serge-30-bn
# #187364839 Set up PostgreSQL (Supabase) as DB Engine with Sequelize ORM

#### What does this PR do?

This PR integrates PostgreSQL with our Express application to provide database persistence for user data.

#### Description of Task to be completed?

*Configures Sequelize ORM to connect Express with a PostgreSQL database.
*Sets up initial migrations and models for managing user data.
*Implements CRUD operations for user entities in the Express routes for testing.

#### How should this be manually tested?

*Start the Express server and ensure it successfully connects to the PostgreSQL database.
*Manually test each CRUD operation (create, read, update, delete) for user entities using tools like Postman or Thunderclient
*you can edit the schema by applying migrations

#### What are the relevant pivotal tracker/Trello stories?

#187364838
